frontend/src/components/regions/CardDetails/utils.js

```diff
@@ -17,6 +18,6 @@ export function canSetDueTime({ card, user, authUser }) {
     if(card.dueTime) {
         return false;
     } else {
-        return user.id === authUser.userId || (user.id !== authUser.userId  && hasPermission({authUser, user}))
+        return authUser.userId || (user.id !== authUser.userId  && hasPermission({authUser, user}))
```

Comment:
This seems incorrect. Basically you are saying that ANY authenticated user can set the due time. 

This is not the functionality we need.

If a due time is not set then a user can set their own due time. 

A user with managment permission can set the due time whether t is set or not.

Reply:
Pseudocode:

```
if it's my card AND due time is empty: return True 

if I have manage_cards permission then return True 

Otherwise return False
```